src/main/java/teammates/common/datatransfer/CourseAttributes.java

```diff
@@ -60,6 +60,10 @@ public String getName() {
         return name;
     }
     
+    public boolean isArchived() {
+        return isArchived;
+    }
+    
```

Comment:
Not the reviewer, but can you ignore the isArchived attribute on CourseAttributes? It will actually be removed soon